src/main/java/nextstep/subway/line/domain/Line.java

```diff
@@ -1,23 +1,24 @@
 package nextstep.subway.line.domain;
 
 import nextstep.subway.BaseEntity;
+import nextstep.subway.line.dto.LineRequest;
 import nextstep.subway.station.domain.Station;
 
 import javax.persistence.*;
-import java.util.ArrayList;
 import java.util.List;
 
 @Entity
 public class Line extends BaseEntity {
+
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     @Column(unique = true)
```

Comment:
```suggestion

    @Column(unique = true)
```
가독성을 위해 한줄씩 띄워주면 좋을 것 같습니다 :)